### PR TITLE
[Merged by Bors] - Fix systemtests to match new logs (broken by #2296)

### PR DESCRIPTION
**Pull request by lrettig**
## Motivation
#2296 broke system tests (see #2328). This fixes them, by updating them to look for the new log format.

## Changes
Update the logs that system tests are looking for to match the new format

## Test Plan
N/A

### CODE REVIEW

**Comment by lrettig** [comment]
bors try

**Comment by lrettig** [comment]
bors try

**Comment by lrettig** [comment]
bors try

**Comment by lrettig** [comment]
bors try

File: tests/p2p/test_p2p.py [hunk 1]
```diff
@@ -21,15 +21,14 @@
...
 
 def query_bootstrap_es(namespace, bootstrap_po_name):
-    hits = poll_query_message(current_index, namespace, bootstrap_po_name, {"M": "Local node identity"}, expected=1)
+    hits = poll_query_message(current_index, namespace, bootstrap_po_name, {"M": "local node identity"}, expected=1)
     for h in hits:
-        match = re.search(r"Local node identity >> (?P<bootstrap_key>\w+)", h.M)
-        if match:
-            return match.group('bootstrap_key')
+        if getattr(h, 'key', None):
+            return h.key
```
Comment (noamnelke) [inline tests/p2p/test_p2p.py:31]: No biggie, but:
```python
try:
    return h.key
```
would be more idiomatic python, IMO.

**Review by noamnelke — APPROVED** [review]
bors merge

**Comment by noamnelke** [comment]
bors merge

**Pull request closed without merging**